changelog_unreleased/mdx/11563.md

```diff
@@ -1,21 +1,16 @@
-#### Add support for MDX 2 comments (#11563 by @wooorm)
+#### [HIGHLIGHT]Add support for MDX 2 comments (#11563 by @wooorm)
 
-This adds basic support for MDX 2 comments (JavaScript-style comments) in
-addition to existing MDX 1 comments (HTML-style comments).
+This adds basic support for MDX 2 comments (JavaScript-style comments) in addition to existing MDX 1 comments (HTML-style comments).
 
-Note that in MDX 2, more complex comments and whole expressions can be used
-inside braces.
-This currently only supports a single comment on one line so that
-`{/* prettier-ignore */}` can be used, but doesn’t support most of MDX 2.
+Note that in MDX 2, more complex comments and whole expressions can be used inside braces.
+This currently only supports a single comment on one line so that `{/* prettier-ignore */}` can be used, but doesn’t support most of MDX 2.
```

Comment:
```suggestion
**Note**: Prettier currently only supports the new comment syntax for single-line comments (so that `{/* prettier-ignore */}` can be used), and doesn’t support the rest of MDX v2.
```